apps/web/nuxt.config.ts

```diff
@@ -63,6 +63,7 @@ export default defineNuxtConfig({
       storename: process.env.STORENAME || 'PLENTYSYSTEMS AG',
       noCache: process.env.NO_CACHE || '',
       configId: process.env.CONFIG_ID || '',
+      isHero: process.env.IS_HERO === 'true',
```

Comment:
```suggestion
      isHero: 'true',
```
This can still be overwritten via `NUXT_PUBLIC_IS_HERO=false`

Reply:
Sorry, I don't understand what you mean